Documentation/2.0/runtime-configuration.md

```diff
@@ -2,6 +2,8 @@
 
 etcd comes with support for incremental runtime reconfiguration, which allows users to update the membership of the cluster at run time.
 
+The reconfiguration request can only be processed when the the majority of the cluster members are functioning. It is also **highly not recommended** to remove a member from a two member cluster. The majority of a two member cluster is also two. If there is a failure during the removal process, the cluster might not be recovered.
```

Comment:
Highly not recommended is kind of hard to understand. I first read this as "highly recommended". What about this:

"When reconfiguring a 2 member cluster down to a single member, any failure during the removal process may leave the cluster in an unrecoverable state. This may happen because the quorum of the cluster is equal to its size."